packages/dnb-eufemia/src/extensions/forms/Field/Upload/Upload.tsx

```diff
@@ -117,6 +117,12 @@ function UploadComponent(props: Props) {
 
   const { files: fileContext, setFiles } = useUpload(id)
 
+  const filesRef = useRef(null)
```

Comment:
```suggestion
  const filesRef = useRef<Array<UploadFile>>()
```

When we define the expected type, we don't need a default value.